pkg/master/master.go

```diff
@@ -840,17 +840,18 @@ func (m *Master) experimental(c *Config) *apiserver.APIGroupVersion {
 	}
 
 	return &apiserver.APIGroupVersion{
-		Root: m.apiGroupPrefix + "/" + latest.GroupOrDie("experimental").Group,
+		Root: m.apiGroupPrefix,
 
 		Creater:   api.Scheme,
 		Convertor: api.Scheme,
 		Typer:     api.Scheme,
 
-		Mapper:  latest.GroupOrDie("experimental").RESTMapper,
-		Codec:   latest.GroupOrDie("experimental").Codec,
-		Linker:  latest.GroupOrDie("experimental").SelfLinker,
-		Storage: storage,
-		Version: latest.GroupOrDie("experimental").Version,
+		Mapper:        latest.GroupOrDie("experimental").RESTMapper,
```

Comment:
Nit: suggest an `expMeta := latest.GroupOrDie("experimental")` instead of repeated calls.